[2025-07-07T19:46:10.162+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_activity_analysis.load_data manual__2025-07-07T19:45:56.179762+00:00 [queued]>
[2025-07-07T19:46:10.168+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_activity_analysis.load_data manual__2025-07-07T19:45:56.179762+00:00 [queued]>
[2025-07-07T19:46:10.168+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2025-07-07T19:46:10.184+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): load_data> on 2025-07-07 19:45:56.179762+00:00
[2025-07-07T19:46:10.193+0000] {standard_task_runner.py:57} INFO - Started process 1554 to run task
[2025-07-07T19:46:10.196+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'customer_activity_analysis', 'load_data', 'manual__2025-07-07T19:45:56.179762+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/customer_activity_dag.py', '--cfg-path', '/tmp/tmp02ybj_xb']
[2025-07-07T19:46:10.198+0000] {standard_task_runner.py:85} INFO - Job 51: Subtask load_data
[2025-07-07T19:46:10.213+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-07T19:46:10.243+0000] {task_command.py:410} INFO - Running <TaskInstance: customer_activity_analysis.load_data manual__2025-07-07T19:45:56.179762+00:00 [running]> on host 7a17032e36f4
[2025-07-07T19:46:10.465+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ds_const@mail.ru' AIRFLOW_CTX_DAG_OWNER='Daniil' AIRFLOW_CTX_DAG_ID='customer_activity_analysis' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-07T19:45:56.179762+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-07T19:45:56.179762+00:00'
[2025-07-07T19:46:10.884+0000] {logging_mixin.py:150} INFO - Successfully saved 220134 records to /opt/***/data/flags_activity.csv
[2025-07-07T19:46:10.886+0000] {logging_mixin.py:150} INFO - Файл /opt/***/shared_tmp/transform_manual__2025-07-07T19:45:56.179762+00:00.csv успешно удалён
[2025-07-07T19:46:10.886+0000] {logging_mixin.py:150} INFO - Temporary file /opt/***/shared_tmp/transform_manual__2025-07-07T19:45:56.179762+00:00.csv removed
[2025-07-07T19:46:10.889+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-07-07T19:46:10.897+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=customer_activity_analysis, task_id=load_data, execution_date=20250707T194556, start_date=20250707T194610, end_date=20250707T194610
[2025-07-07T19:46:10.928+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-07-07T19:46:10.943+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
